.github/workflows/check.yaml

```diff
@@ -40,4 +40,12 @@ jobs:
       - name: Show contributors on all branches
         run: git shortlog -s -n --all --no-merges
 
+      - name: clean for release build
+        run: make clean
+
+      - name: qmake
+        run: qmake game_on_gha.pro
+
```

Comment:
I don't know yet what this means but the build passes so I trust your judgement. 🌞